### PR TITLE
[BREAKING CHANGE] Edit Database Schema

### PR DESCRIPTION
**Pull request by PKhing**
## What did you do

- change `isVerified` to `status`
- move `description` from `Actor` to `User`
- add `updatedAt` to `User`
- remove `bodyModifications`

## Demo

- merged to dev (please seed)

### CODE REVIEW

**Review by palmcm — APPROVED** [review]

File: apps/web/src/common/hoc/withGuard/index.tsx [hunk 1]
```diff
@@ -18,7 +19,7 @@ const withGuard = (
       return null
     }
 
-    if (!user.isVerified) {
+    if (user.status !== UserStatus.ACCEPTED) {
```
Comment (miello) [inline apps/web/src/common/hoc/withGuard/index.tsx:22]: Should this have 2 cases ?
- REJECTED
- PENDING

**Review by miello — CHANGES_REQUESTED** [review]
Wait, what is `tsconfig.tsbuildinfo` ?

File: apps/web/src/common/hoc/withGuard/index.tsx (hunk 1 above)
Comment (PKhing) [reply apps/web/src/common/hoc/withGuard/index.tsx:22]: will be fixed when reject page is implemented

**Comment by PKhing** [comment]
> Wait, what is `tsconfig.tsbuildinfo` ?

It might come from `pnpm tsc` 😂 Already removed

**Review by miello — APPROVED** [review]

**Pull request merged**